js/stats.js

```diff
@@ -44,6 +44,19 @@ const logPodStats = () => {
         });
 };
 
+/**
+ * ELECTRON-956: App is not minimized upon "Configure Desktop Alert Position" modal when "Always on Top" = True
+ */
+app.on('browser-window-created', (event, window) => {
```

Comment:
move this logic to window manager file. Doesn't make sense to have it in stats.js